alert/alert.css

```diff
@@ -0,0 +1,60 @@
+@import url(../styles.css);
+
+html {
```

Comment:
Suggestion:
If you're using common style file, you can even move these styles to common file.

Reply:
I am thinking of doing this after creating all of the components and then putting the common CSS to style.css